### ISSUE TITLE
Fine tuned to Multi-choice dataset?

### ISSUE THREAD

**Issue by Qzsl123** [huggingface/pytorch-pretrained-BERT#90]
Is it posible to fine tuned to the multi choices problems , which usually has one passage, question and ABCD four options?

**Comment by Qzsl123** [comment]
Thx for the reply.
Actually, I have the same plan. But I am not sure whether it will work. Anyway, I will have a try.

**Comment by rodgzilla** [comment]
If it worked in the OpenAI paper, I don't really see why it wouldn't work with this architecture.

**Comment by Qzsl123** [comment]
@rodgzilla yeah, I am trying to run it. Thanks for the wonderful job!

**Comment by DukeZhu** [comment]
> Yes it is, the code is not written yet but I'm planning to work on it. The idea is to format the input data the same way the authors of [Improving Language Understanding with Unsupervised Learning](https://blog.openai.com/language-unsupervised/)
> 
> ![Multiple choice GPT](https://camo.githubusercontent.com/e5d95abc42ca2acb493a710383c949eb01c10bfb/68747470733a2f2f692e696d6775722e636f6d2f7a3045616e76792e706e67)
> 
> You run an inference `(context, choice)` for each choice, you compute the image of the `[CLS]` token by a linear layer with 1 output and then compute a softmax over the output of all choices.
> 
> I will try to create a PR with this code very soon.

hi，The multi choices problem usually has one passage, question and ABCD four options。In your model, dose context means passage&question ?

**Comment by monk1337** [comment]
Any update on this issue?
